boards/renesas/rzg3s_smarc/rzg3s_smarc_r9a08g045s33gbg_cm33_defconfig

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 EPAM Systems
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SYS_CLOCK_EXISTS=y
+
+CONFIG_XIP=n
+
+CONFIG_MAIN_STACK_SIZE=4096
+
```

Comment:
```suggestion
```

Reply:
I agree 4096 looks a bit high.
@binhnguyen2434 can you explain why so much main stack is needed ?

Reply:
I verified with default stack 1024 of system. That's work fine,
So, this config is unnecessary, I removed it.